charts/nexus-repository-manager/README.md

```diff
@@ -1,8 +1,8 @@
-# Nexus
+# Nexus Repository
 
-[Nexus OSS](https://www.sonatype.com/nexus-repository-oss) provides universal support for all major build tools.
+[Nexus Repository OSS](https://www.sonatype.com/nexus-repository-oss) provides universal support for all major build tools.
```

Comment:
NOTE: Not sure the "OSS" is needed here, tho that is where the link goes :)  Since no-one asked me to do this I left this alone ;)